package-lock.json

```diff
@@ -310,7 +321,7 @@
     },
     "ansi-escapes": {
       "version": "1.4.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-1.4.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-1.4.0.tgz",
```

Comment:
Hmm, any idea what happened here? Seems odd to switch to an unencrypted connection ...

Reply:
npm keeps flipping them and I don't know why :(

Reply:
(the overall diff doesn't change the file, so not an issue)